src/pretix/base/exporters/waitinglist.py

```diff
@@ -0,0 +1,155 @@
+
+from collections import OrderedDict
+
+from django import forms
+from django.db.models import F, Q
+from django.dispatch import receiver
+from django.utils.timezone import now
+from django.utils.translation import gettext_lazy as _, pgettext_lazy
+
+from pretix.base.models.waitinglist import WaitingListEntry
+
+from ..exporter import ListExporter
+from ..signals import (
+    register_data_exporters, register_multievent_data_exporters,
+)
+
+
+class WaitingListExporter(ListExporter):
+    identifier = 'waitinglist'
+    verbose_name = _('Waiting list')
+
+    # map selected status to label and queryset-filter
+    status_filters = [
+        (
+            '',
+            _('All entries'),
+            lambda qs: qs
+        ),
+        (
+            'awaiting-voucher',
+            _('Waiting for a voucher'),
+            lambda qs: qs.filter(voucher__isnull=True)
```

Comment:
Fine for now, but as a suggestion for future similar things, you could use [Q objects](https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.Q) instead of lambdas, e.g. ``Q(voucher__isnull=True)``

Reply:
In this case, as it always applies .filter() on the queryset, Q objects are an option. But I originally considered using .exclude() on 'voucher-assigned' (don’t know why it felt semantically better to me). Furthermore using the queryset and lambdas makes it possible to add further manipulations to the queryset beyond just a filter – e.g. sort differently, group, etc. Currently not needed, but I think it is more flexible this way.

Reply:
Understood :) ``exclude()`` is still possible with Q objects (``~Q(excluded…)``), but ``annotate`` e.g. would not be, correct. Let's keep it like this for now :)